src/Equinor.ProCoSys.IPO.Command/InvitationCommands/EditParticipants/EditParticipantsCommandValidator.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.Command.InvitationCommands.EditInvitation;
+using Equinor.ProCoSys.IPO.Command.Validators.InvitationValidators;
+using Equinor.ProCoSys.IPO.Command.Validators.RowVersionValidators;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using FluentValidation;
+
+namespace Equinor.ProCoSys.IPO.Command.InvitationCommands.EditParticipants
+{
+    public class EditParticipantsCommandValidator : AbstractValidator<EditParticipantsCommand>
+    {
+        public EditParticipantsCommandValidator(IInvitationValidator invitationValidator, IRowVersionValidator rowVersionValidator)
+        {
+            CascadeMode = CascadeMode.Stop;
+
+            RuleFor(command => command)
+                //input validators
+                .Must(command => command.UpdatedParticipants != null)
```

Comment:
Should also check on .Any()?

Reply:
This will be caught in TwoFirstParticipantsMustBeSetWithCorrectOrganization, where we check that participants.Count < 2, is it still good to add the .Any() ? Without the error message to the user is "Contractor and Construction Company must be invited!"

Reply:
Easier to read code if you add Any. Change message to "Participants must be invited!"